### PR TITLE
[SD-3388] - Fetching should not add byline and dateline

### PR DESCRIPTION
**Pull request by akintolga**

### CODE REVIEW

File: server/apps/archive/common.py [hunk 1]
```diff
@@ -537,25 +540,26 @@ def copy_metadata_from_user_preferences(doc, repo_type=ARCHIVE):
...
+                if user and user.get('user_preferences', {}).get('dateline:located'):
+                    located = user.get('user_preferences', {}).get('dateline:located', {}).get('located')
+                    if located:
+                        doc['dateline']['located'] = located
+                        doc['dateline']['text'] = format_dateline_to_locmmmddsrc(located, dateline_ts)
 
-        if BYLINE not in doc and user and user.get(BYLINE):
-                doc[BYLINE] = user[BYLINE]
+            if BYLINE not in doc and user and user.get(BYLINE):
+                    doc[BYLINE] = user[BYLINE]
```
Comment (sivakuna-aap) [inline server/apps/archive/common.py]: Looks like this line is over-indented

**Comment by sivakuna-aap** [comment]
:+1:

File: server/apps/archive/common.py (hunk 1 above)
Comment (akintolga) [inline server/apps/archive/common.py]: I think it is due to extra if statement on line 543.

**Comment by mdhaman** [comment]
:+1:

**Comment by mugurrus** [comment]
:+1:

**Pull request merged**